### DISCUSSION TITLE
Keybinding(s) for highlight items and toggle always on / off all dropped items?

### DISCUSSION THREAD

**Discussion by WithinAmnesia** [flareteam/flare-game#856] [Ideas]
Keybinding(s) for highlight items and toggle always on / off all dropped items? Is there a way for a key bind to toggle show all items always on / off? Also potentially an option to just tap / hold a key-bind to temporary show highlighted items on the ground if toggled off? This can avoid issues like missing ground items near an exit or high movement speed / forgetting hard to see items / non-obvious items that did not show for very long on the ground if at all.

**Comment by dorkster** [comment]
The Alt key can temporarily change loot tooltip visibility. If the loot tooltips visibility setting is set to "Show All", pressing Alt will hide all loot tooltips. Otherwise, it will show any hidden loot tooltips.

Since Alt is hidden in the keybindings list like the other modifier keys, there's no way to map this action to another key. So it may be worth creating a user-definable keybind for this action, and possibly a keybind for cycling through the loot tooltip visibility modes.

**Reply by WithinAmnesia** [reply]
Okay that sounds good, didn't you want better game-pad / controller support for Flare? Wouldn't a user defined loot highlight toggle / button work towards this? Different controllers can have more or less buttons so having the ability to map this as a user made keybind would help in many ways.

**Reply by dorkster** [reply]
Yes, adding a new binding will allow gamepads to have access to this action.
